<launch>
  <node pkg="imu" name="ahrs_driver" type="ahrs_driver" output="screen" >
    <!-- 是否输出debug信息 -->
    <param name="debug"  value="false"/>
    
    <!-- 串口设备，可通过rules.d配置固定 -->
    <param name="port"  value="/dev/fdilink_imu"/>
    <!-- 波特率 -->
    <param name="baud"  value="921600"/>

    <!-- 发布的imu话题名 -->
   <!-- <param name="imu_topic"  value="/imu_data"/> -->
    <!-- <param name="imu_topic"  value="/imu_data1"/>-->
    
    <!-- 发布的imu话题中的frame_id -->
    <param name="imu_frame"  value="imu_link"/>

    <!-- 地磁北的yaw角 -->
    <param name="mag_pose_2d_topic"  value="/mag_pose_2d"/>

    <!-- 发布的数据基于不同设备有不同的坐标系   -->
    <param name="device_type"  value="1"/> <!-- 0: origin_data, 1: for single imu or ucar in ROS, 2:for Xiao in ROS -->
  </node>
  <!-- sukai   
  <node name="base_to_imu_link" pk="tf" type="static_transform_publisher" args="0 0 0 0 0 0 /base_link /imu_link"/>
    -->

  <!-- <include file='$(find imu)/launch/tf.launch' /> -->
</launch> 
